config/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+
+// Wrapper around require to set relative path at app root
+global.rootRequire = function (name) {
+  return require(`../${name}`);
+};
```

Comment:
This function is from before my time, my $0.02 is eventually remove it from the repo.